### PR TITLE
fix: can't open lvim log file cause by `lvim.log.viewer.cmd`

### PR DESCRIPTION
**Pull request by younger-1**
When I try to open log file by pressing `<leader>Lld`, nothing happens.

I found it was caused by this line:

https://github.com/LunarVim/LunarVim/blob/238bbf18ac4e35152be06736c75e94a81e3382e8/lua/config/defaults.lua#L23

`lvim.log.viewer.cmd` is not executable in this case. Use `less` instead of `lnav` will solve it

### CODE REVIEW

**Comment by abzcoding** [comment]
cc @kylo252

File: lua/config/defaults.lua [hunk 1]
```diff
@@ -20,7 +20,7 @@ lvim = {
     level = "warn",
     viewer = {
       ---@usage this will fallback on "less +F" if not found
-      cmd = "lnav",
+      cmd = "less",
```
Comment (kylo252) [inline lua/config/defaults.lua:23]: This is not the correct place to change it. 

There's supposed to be a fallback in `toggle_log_viewer` but it disappeared somehow. It should also be `less +F` or `tail -f`.

**Review by kylo252 — CHANGES_REQUESTED** [review]

File: lua/config/defaults.lua [hunk 2]
```diff
@@ -20,7 +20,7 @@ lvim = {
     level = "warn",
     viewer = {
       ---@usage this will fallback on "less +F" if not found
-      cmd = "lnav",
+      cmd = "less +F",
```
Comment (kylo252) [inline lua/config/defaults.lua:23]: Sorry I didn't make it clear. It's better to set it like this:

```lua
local log_viewer = lvim.log.viewer.cmd
if vim.fn.executable(log_viewer) ~= 0 then
	log_viewer = 'less +F'
end
local cmd = log_viewer .. " " .. logfile
local term_opts = vim.tbl_deep_extend("force", lvim.builtin.terminal, {
    cmd = cmd,
	--- rest of the arguments
```

in here:
https://github.com/LunarVim/LunarVim/blob/6b79e00cb04bf889fc30b84482d6db6e951ca9e9/lua/core/terminal.lua#L115-L116

**Review by kylo252 — CHANGES_REQUESTED** [review]

File: lua/config/defaults.lua (hunk 2 above)
Comment (younger-1) [reply lua/config/defaults.lua]: Sorry for my silly mistake. Solved now.

**Comment by kylo252** [comment]
Thank you!

**Pull request merged**